### PR TITLE
Make it possible to attach control-plane logs to stdout

### PR DESCRIPTION
**Pull request by alenkacz**
This is useful when for debugging tests or test setup.

### CODE REVIEW

File: pkg/apis/testharness/v1beta1/test_types.go [hunk 1]
```diff
@@ -25,6 +25,9 @@ type TestSuite struct {
 	// ControlPlaneArgs defaults to APIServerDefaultArgs from controller-runtime pkg/internal/testing/integration/internal/apiserver.go
 	// this allows for control over the args, however these are not serialized from a TestSuite.yaml
 	ControlPlaneArgs []string
+	// AttachControlPlaneOutput if true, attaches control plane logs (api-server, etcd) into stdout. This is useful for debugging.
+	// defaults to false
+	AttachControlPlaneOutput bool
```
Comment (ANeumann82) [inline pkg/apis/testharness/v1beta1/test_types.go:30]:
```suggestion
	AttachControlPlaneOutput bool `json:"attachControlPlaneOutput"`
```

This should have a json mapping, right? I'm not sure why the controlPlaneArgs above don't have it...

**Review by ANeumann82 — APPROVED** [review]
lgtm, apart from the missing json mapping

File: pkg/apis/testharness/v1beta1/test_types.go (hunk 1 above)
Comment (alenkacz) [reply pkg/apis/testharness/v1beta1/test_types.go]: thanks! I've fixed it and added it to the above one as well

**Review by kensipe — APPROVED** [review]
This looks great!   thank!

One small thing (not required for merge)... all configurations are configurable outside of the kuttl config file.  There should be a flag for this. 

The kuttl-test.yaml config file is a convenience file but shouldn't be required.

**Pull request merged**